_partials/wsl2_prereq_win_version.md

```diff
@@ -0,0 +1,64 @@
+### Latest version of Windows
+
+Open Windows Update:
+
+
+>\- Press `Windows` + `R`  
+>\- Type  `ms-settings:windowsupdate`  
+>\- Press `Return`
+>
+>
+>\- Click on `Check updates`
+
+&nbsp;
+
+&nbsp;&nbsp;&nbsp; :x: If you have an error message about Windows not being able to apply updates, please **contact a teacher**.
+
+&nbsp;&nbsp;&nbsp; :white_check_mark: If you have updates available please install them and repeat the process until it says that you are up to date :star:.
+
+
+---
+
+#### :wrench: Potential Fixes
+
+<details>
+  <summary>Activate Windows Update Service to fix Updates</summary>
+
+  &nbsp;
+  
+  
+  Some antiviruses and pieces of software desactivate the Update service we need, resulting in the error you see. Let's fix that!
+
+
+  >\- Press `Windows` + `R`  
+  >\- Type  `services.msc`  
+  >\- Press `Return`
+  >
+  >
+  >\- Double Click `Windows Update Service`  
+  >\- Set its `Startup` to `Automatic`  
+  >\- Click on `Start`  
+  >\- Click on `Ok`
+
+  Then let's try updates again!
+</details>
+
+---
+
+### Minimum version
+
+Some of the tools we need to install have been release with the `1903` version **or above** of Windows 10 so we need to make sure you have at least this one.
```

Comment:
I think it should be version `2004`

Reply:
Microsoft changed the requirements by back-porting WSL2 to earlier versions: [here](https://devblogs.microsoft.com/commandline/wsl-2-support-is-coming-to-windows-10-versions-1903-and-1909/)

Reply:
Didn't know it was active yet, top stuff 👍